rocketmq-iot-bridge/pom.xml

```diff
@@ -64,6 +64,11 @@ limitations under the License.
             <artifactId>slf4j-api</artifactId>
             <version>1.7.7</version>
         </dependency>
+        <dependency>
```

Comment:
Pls do not import the slf4j implementation, provided or optional scope are recommended.

Reply:
I've use logback-classic to resolve this.